### DISCUSSION TITLE
config for setup Multi Clusters

### DISCUSSION THREAD

**Discussion by KoGJ95** [ClickHouse/ClickHouse#52348] [Q&A]
Hi all.
Before asking something, I'm sorry that I'm not using English as a first language, so not good at English.

I have question with configuration for multi clusters.
I read article in ChistaData about setup cluster and zookeeper and have some question.
Link : https://chistadata.com/setup-clickhouse-cluster-replication-with-zookeeper/

They wrote about distributed_ddl parameter in zookeeper.xml, and saying "<distributed_ddl><path>: This needs to be unique for each cluster. We’re only creating one cluster with our 2  DB nodes."
If I made 2 clusters in 1 server, then how can I make that parameter correctly?

Have a good day everyone!

**Comment by zztaoli** [comment]
You can set it like this:
cluster_1：

    <distributed_ddl>
    	<path>/clickhouse/cluster_1/task_queue/ddl</path>
    </distributed_ddl>`

cluster_2:

    <distributed_ddl>
    	<path>/clickhouse/cluster_2/task_queue/ddl</path>
    </distributed_ddl>
Each cluster has its own zk directory path

**Reply by KoGJ95** [reply]
Thank you! But I am using a total of 4 servers, with only one ClickHouse installed on each server, so I have only one set of XML data per server.
